integration/plugin_manager/plugin_install_spec.rb

```diff
@@ -0,0 +1,43 @@
+# Encoding: utf-8
+require_relative "../spec_helper"
+require "fileutils"
+
+context "bin/plugin install" do
+  context "with a local gem" do
+    let(:gem_name) { "logstash-input-wmi" }
+    let(:local_gem) { gem_fetch(gem_name) }
+
+    it "install the gem succesfully" do
+      result = command("bin/plugin install --no-verify #{local_gem}")
+      expect(result.exit_status).to eq(0)
+      expect(result.stdout).to match(/Installing #{gem_name}/)
```

Comment:
I wonder if it makes sense to do this matches all in one line? so we track also the text format? what do you think?